src/components/atoms/Checkbox/index.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import classNames from 'classnames'
+
+import Input from '_atoms/Input'
+
+import styles from './styles.css'
+
+const Checkbox = props => {
+  const { className, isChecked } = props
+  const inputClasses = classNames({
+    [className]: true,
+    [styles.input]: true,
+    [styles.checked]: isChecked,
+  })
+
+  return (
+    <label className={styles.checkbox} role="checkbox" aria-checked={isChecked}>
```

Comment:
Is the label necessary? It should be used to add labels to fields, but there's no label here.